test/test_autograd.py

```diff
@@ -4877,18 +4880,22 @@ def gradgradcheck_method_precision_override(test_name):
     return override
 
 def run_grad_and_gradgrad_checks(test_case, name, test_name, apply_method, output_variable,
-                                 input_variables, run_gradgradcheck=True):
-    test_case.assertTrue(gradcheck(apply_method, input_variables, eps=1e-6, atol=PRECISION))
+                                 input_variables, run_gradgradcheck=True, check_batched_grad=True):
```

Comment:
Tests using method_tests() are being deprecated in favor of tests using OpInfos like these:

https://github.com/pytorch/pytorch/blob/bd322c8967b99551df67d479ebcc435c030980c9/test/test_ops.py#L57

I'm developing a post on this now, but the plan is to remove method_tests(). This PR doesn't have to update the OpInfo-based testing but that should happen in the near future to continue testing this property. Instead of a custom `EXCLUDE_BATCHED_GRAD_TESTS` a property could be added to the OpInfo class for whether to run batched grad tests or not, or the OpInfo skip mechanism can be reused.

Reply:
I'm happy to add something to the OpInfo class in a followup to make sure these are tested in the future.